includes/base/autoloader-base.php

```diff
@@ -0,0 +1,195 @@
+<?php
+namespace Elementor;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Elementor autoloader.
+ *
+ * Elementor autoloader handler class is responsible for loading the different
+ * classes needed to run the plugin.
+ *
+ * @since 1.6.0
+ */
+class Autoloader_Base {
+
+	/**
+	 * Classes map.
+	 *
+	 * Maps Elementor classes to file names.
+	 *
+	 * @since 1.6.0
+	 * @access private
+	 * @static
+	 *
+	 * @var array Classes used by elementor.
+	 */
+	protected static $classes_map;
```

Comment:
Move classes map to `includes/autoloader.php`

Reply:
Handled.